CSS/map.css

```diff
@@ -4,4 +4,5 @@
     background-color: #ddd;
```

Comment:
Same comment as above.   A bit more comments on what this file is doing would be useful.